Common/Data/Affixes/WeaponAffixes.json

```diff
@@ -335,5 +335,36 @@
 				"weight": 1
 			}
 		]
+	},
+	{
+		"affixType": "ChanceToApplyRootedGearAffix",
+		"equipTypes": "Weapon",
+		"influences": "None",
+		"tiers": [
+			{
+				"minValue": 1,
+				"maxValue": 4,
+				"minimumLevel": 1,
+				"weight": 1
+			},
+			{
+				"minValue": 5,
+				"maxValue": 8,
+				"minimumLevel": 21,
+				"weight": 1
+			},
+			{
+				"minValue": 9,
+				"maxValue": 12,
+				"minimumLevel": 52,
+				"weight": 1
+			},
+			{
+				"minValue": 13,
+				"maxValue": 16,
+				"minimumLevel": 73,
+				"weight": 1
+			}
+		]
```

Comment:
Can remove this entirely. We want this affix to only exist on uniques at this time.